content/fact/arc4random.md

```diff
@@ -0,0 +1,17 @@
+---
+title: arc4random(3)
+---
+
+arc4random: the cryptographic random number generator.
+
+Arc4random is "A Replacement Call For Random", to generate very quickly high
+ quality 32-bit pseudo-random numbers.
+
+Arc4random first appeared in OpenBSD 2.1. In OpenBSD 5.5, it was replaced with
+ the ChaCha20 cipher!
```

Comment:
"OpenBSD has its own cryptographic random number generator.

Whereever random input is needed, `arc4random` is used. `arc4random` is an abstraction layer for currently considered as safe ciphers and produces ChaCha20 ciphers at the moment"

What do you think?

Reply:
I think: it's a good measure to harden the generation. Using ChaCha20 ciphers seems to be a seriously choice because is a robust actually safe cipher. But I'm not developer... just a little IT!